src/main/java/ro/fortsoft/pf4j/update/DefaultUpdateRepository.java

```diff
@@ -51,7 +51,7 @@ public String getId() {
     }
 
     @Override
-    public String getLocation() {
+    public String getUrl() {
```

Comment:
Also in constructor and [class field](https://github.com/cominvent/pf4j-update/blob/0f7cdc1a9aca7ab66ba5170d7648df570e5573ae/src/main/java/ro/fortsoft/pf4j/update/DefaultUpdateRepository.java#L38). In this mode we eliminate the lines with `new URL(url)` from the `initPlugins` method.